certified-connectors/Dokobit Universal API/readme.md

```diff
@@ -0,0 +1,14 @@
+## Dokobit Universal API
```

Comment:
@epelanis please include all required sections in the readme file. Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).